shell.nix

```diff
@@ -1,26 +1,18 @@
-{ pkgs ? import (fetchGit {
-    url = https://github.com/dapphub/dapptools.git;
-    rev = "6943c76bfb8e0b1fce54c3d9bba6f0f7e50d2f5c";
-  }) {}
-, mcd-cli ? pkgs.callPackage (import (fetchGit {
-    url = https://github.com/makerdao/mcd-cli.git;
-    rev = "86842b49defa53301ac0019f7d5994859bb3e1e9";
-  })) {}
-, sethret ? (import (fetchGit {
-    url = https://github.com/icetan/sethret.git;
-    rev = "ef77915e2881011603491275f36b44bf2478b408";
-  }) {}).sethret
-}:
+{ pkgsSrc ? (import ./nix/pkgs.nix {}).pkgsSrc
+, pkgs ? (import ./nix/pkgs.nix { inherit pkgsSrc; }).pkgs
```

Comment:
What's the point of passing `inherit pkgsSrc` here?

Reply:
I you change the `nixpkgs` source it needs to pass through the `nixpkgs-pin` expression as well.

Reply:
Ooooh, now I understand! The `inherit pkgsSrc` evaluated in the context of `nixpkgs-pin`'s `default.nix`.

I thought it was affecting the import inside `nix/pkgs.nix`.

All good, thanks!